etl/steps/data/garden/who/2024-04-08/polio.py

```diff
@@ -0,0 +1,326 @@
+        "Mauritania",
+        "Namibia",
+        "Rwanda",
+        "Tanzania",
+        "Zimbabwe",
+        "Lebanon",
+        "Bangladesh",
+        "India",
+        "East Timor",
+        "Bolivia",
+        "Cambodia",
+        "Malaysia",
+    ]
+
+    difference = [item for item in higher_risk_entities if item not in tb["country"].unique()]
+    assert difference == [], f"Entities in the high-risk list that are not in the dataset: {difference}"
+
+    # Define the condition for which countries are not in high-risk entities
+    not_high_risk = ~tb["country"].isin(higher_risk_entities)
+
+    # Define the condition for screening year
+    is_screening_year = tb["year"] == SCREENING_YEAR
+
+    # Combine conditions and update 'polio_surveillance_status' for matching rows
+    tb.loc[not_high_risk & is_screening_year, "polio_surveillance_status"] = "Low risk"
+
+    return tb
+
+
+def add_screening_and_testing(tb: Table, tb_regions: Dataset, ds_regions: Dataset) -> Table:
+    """
+    Adds the polio surveillance status based on the screening and testing rates.
+    For use in this chart: https://ourworldindata.org/grapher/polio-screening-and-testing
+
+    Parameters:
+    - tb: table containing polio surveillance data.
+    - year: Specific year to filter the data. If None, uses current year.
+
+    Returns:
+    - Modified table with a new column for polio surveillance status.
+    """
+    # Ensuring we have all the countries in the WHO regions - even if there isn't other polio data for them
+    who_countries = list_of_who_countries(tb_regions, ds_regions)
+    who_tb = Table({"country": who_countries, "year": SCREENING_YEAR}).copy_metadata(from_table=tb)
+    tb = tb.merge(who_tb, on=["country", "year"], how="outer")
+
+    # Add the polio surveillance status based on the screening and testing rates
+    tb.loc[
+        (tb["non_polio_afp_rate"] >= 2.0)
+        & (tb["pct_adequate_stool_collection"] >= 80)
+        & (tb["year"] == SCREENING_YEAR),
+        "polio_surveillance_status",
+    ] = "Adequate screening and testing"
+    tb.loc[
+        (tb["non_polio_afp_rate"] >= 2.0) & (tb["pct_adequate_stool_collection"] < 80) & (tb["year"] == SCREENING_YEAR),
+        "polio_surveillance_status",
+    ] = "Inadequate testing"
+    tb.loc[
+        (tb["non_polio_afp_rate"] < 2.0) & (tb["pct_adequate_stool_collection"] >= 80) & (tb["year"] == SCREENING_YEAR),
+        "polio_surveillance_status",
+    ] = "Inadequate screening"
+    tb.loc[
+        (tb["non_polio_afp_rate"] < 2.0) & (tb["pct_adequate_stool_collection"] < 80) & (tb["year"] == SCREENING_YEAR),
+        "polio_surveillance_status",
+    ] = "Inadequate screening and testing"
+
+    tb = identify_low_risk_countries(tb, tb_regions, ds_regions)
+    # Not sure if this is the best way to handle this, the code fails because this indicator doesn't have origins otherwise
+    tb["polio_surveillance_status"].metadata.origins = tb["non_polio_afp_rate"].metadata.origins
```

Comment:
You just created the column `"polio_surveillance_status"` from scratch, so it doesn't have any metadata. You could simply do `tb["polio_surveillance_status"] = tb["polio_surveillance_status"].copy_metadata(tb["non_polio_afp_rate])`.